### PR TITLE
Add remapUVSetsTo option to UsdImport

### PR DESCRIPTION
**Pull request by dj-mcg**
Our users have requested that UV Sets that were renamed on export round trip cleanly on import. The Maya:name metadata accomplishes this, but only for assets that were exported after the introduction of this code. This allows us to specify a remapping for UV Set names on import so older assets work.

If the Maya:name metadata is found, this remapping is not used.

### CODE REVIEW

File: lib/mayaUsd/fileio/utils/readUtil.h [hunk 1]
```diff
@@ -42,6 +42,16 @@ struct UsdMayaReadUtil
     MAYAUSD_CORE_PUBLIC
     static bool ReadFloat2AsUV();
 
+    /// Returns the imported name of a UV set at \p uvIndex according to the renaming rule and the
```
Comment (pierrebai-adsk) [inline lib/mayaUsd/fileio/utils/readUtil.h:45]: This function description does not seem accurate and refer to a wrong parameter name.

**Review by pierrebai-adsk — COMMENTED** [review]
Just a function header comments that does not seem right, otherwise LGTM.

File: lib/mayaUsd/fileio/utils/readUtil.h (hunk 1 above)
Comment (dj-mcg) [reply lib/mayaUsd/fileio/utils/readUtil.h:45]: Oh, excellent catch - I changed the function signature but forgot to change the comment - I'll fix@

**Comment by dj-mcg** [comment]
I fixed the comment in place - thanks for catching!

**Pull request merged**